Resources/public/js/bootstrap.collection.js

```diff
@@ -47,17 +47,18 @@
         newLi.appendTo(list);
 
         // trigger de l'évènement d'ajout
-        $(document).trigger('collection.item.added', list.find('li').last());
+        $this.closest('[id^=sidus_tabbed_data_]').trigger('collection.item.added', list.find('li').last());
     };
 
     CollectionRemove.prototype.removeField = function (e) {
         var $this = $(this),
-            selector = $this.attr('data-field')
-            ;
+          selector = $this.attr('data-field')
+          ;
 
         e && e.preventDefault();
-
+        var $collection = $this.closest('[id^=sidus_tabbed_data_]');
```

Comment:
Pour ce qui est de renvoyer l'objet, 2 questions : 
1. pourquoi ?
2. comment ?
Au moment ou on trigg l'evenement, l'élément est dejà supprimé, et c'est souhaitable, il faut pouvoir compter le nombre d'éléments restants par exemple.
A la limite, on pourrait trigger 2 events un "before.remove" et un "removed" pour couvrir tous les cas d'usage non ? ou alors j'ai pas compris un truc.

Alors le "$", ça ressemble a un fail de dev php fatigué mais il n'en est rien :) C'est une convention pour distinguer un objet jQuery d'un autre variable en js. Comme quand on écrit $this = $(this) finalement.